libsyclinterface/cmake/modules/FindIntelSycl.cmake

```diff
@@ -39,6 +39,48 @@ find_package(IntelDPCPP REQUIRED)
 set(clangxx_cmd "${CMAKE_CXX_COMPILER}")
 set(clangxx_arg "--version")
 
+# Compares the two version string that are supposed to be in x.y.z format
+# and reports if the argument VERSION_STR1 is greater than or equal than
+# version_str2. The strings are compared lexicographically after conversion to
+# lists of equal lengths, with the shorter string getting zero-padded.
+function(compare_versions VERSION_STR1 VERSION_STR2 OUTPUT)
```

Comment:
Perhaps a more suitable name is `versions_greater_equal`.   I would be expecting `compare_versions` to return -1, 0, or 1, like `strcmp` does.

Reply:
Done.